### PR TITLE
Adding the new plugin: basketball

### PR DESCRIPTION
**Pull request by kefthymic**
## Description
I finished the basketball plugin. I tested my code using the test_plugin_script.

### CODE REVIEW

File: docs/source/Plugins/fun.rst [hunk 1]
```diff
@@ -36,6 +36,11 @@ Magic 8 Ball
 .. automodule:: plugins.downloaded.magic_ball.main
    :members:
 
+Basketball
+^^^^^
```
Comment (Abdur-rahmaanJ) [inline docs/source/Plugins/fun.rst:40]: Add the `^^^` to underline the title completely

**Review by Abdur-rahmaanJ — CHANGES_REQUESTED** [review]

**Comment by Abdur-rahmaanJ** [comment]
Great PR else!

File: docs/source/Plugins/fun.rst (hunk 1 above)
Comment (kefthymic) [reply docs/source/Plugins/fun.rst]: I just changed it. I think now it's ok

**Comment by Abdur-rahmaanJ** [comment]
@MrTanoshii Please review

**Comment by Abdur-rahmaanJ** [comment]
hey @kefthymic Can you add it on the README? Thanks!

**Comment by kefthymic** [comment]
I added the plugin on the README. Is it ok?

**Review by MrTanoshii — CHANGES_REQUESTED** [review]
There's a few formatting/style issues with `main.py` and `README.md`

Can you please do the following:
- Run the command `black main.py`
- Format `README.md` using VS Code or a free tool like https://codebeautify.org/markdown-formatter

**Comment by kefthymic** [comment]
@MrTanoshii I formatted the README using the https://codebeautify.org/markdown-formatter as you told me and also ran the command for the main.py. Is there anything I need to do?

**Review by MrTanoshii — APPROVED** [review]
LGTM

**Comment by MrTanoshii** [comment]
Thank you once again!

There's nothing else to do, your plugin is now merged 🥳

**Pull request merged**